test/functional/wallet_importmulti.py

```diff
@@ -625,6 +625,76 @@ def run_test(self):
                      ismine=False,
                      iswatchonly=False)
 
+        # Import some public keys to the keypool of a no privkey wallet
```

Comment:
A test showing that importing some non-single-pubkey addresses using descriptors doesn't cause the keypool to grow seems apt.

Reply:
Done